### PR TITLE
:book: first edit of the docs

### PR DESCRIPTION
**Pull request by amanroa**
## Summary

Did a high level edit of the documents (grammar, formatting, etc).

### CODE REVIEW

**Comment by clubanderson** [comment]
@xrstf can you help here?  Don't know why the job is failing. I don't know what it tests for.

**Comment by xrstf** [comment]
The verify job does

```
    - command:
      - make
      - verify-boilerplate
      - verify-codegen
      - verify-go-versions
      - verify-imports
      - verify-k8s-deps
      - verify-modules
      - verify-syncer-codegen
```

(see .prow.yaml)

You can click on the DETAILS links to the verify job to see the build log: https://public-prow.kcp.k8c.io/view/s3/prow-public-data/pr-logs/pull/kcp-dev_edge-mc/581/pull-edge-mc-verify/1666915974894325760 -- seems like something in the go.mod is borked. Maybe https://github.com/kcp-dev/edge-mc/commit/785b1b49b726f5d1d9090a9d47e846dae1b34508 broke something because it was committed directly to the main branch instead of going through a PR?

This PR also has merge conflicts that need to be resolved and have nothing to do with prow. Prow cannot merge a PR when there are merge conflicts.

> Conflicting files
> docs/content/Coding Milestones/PoC2023q1/kubestellar-scheduler-subs/kubestellar-scheduler-0-pull-kcp-and-kuberstellar-source-and-start-kcp.md

**Comment by clubanderson** [comment]
Thank you @xrstf. I know about the rebase. That's easy. 

@MikeSpreitzer can you have a look at the go mod to see what's wrong with it?  I think the push references above is from your work.

File: docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-start-kcp.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!--example1-start-kcp-start-->
 ### Start kcp
 
-Download and build or install [kcp](https://github.com/kcp-dev/kcp),
+Download and build or install [kcp](https://github.com/kcp-dev/kcp/releases),
```
Comment (waltforme) [inline docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-start-kcp.md:4]: Maybe use one specific release?

**Pull request merged**